src/frontend/query_commands.ml

```diff
@@ -502,6 +502,15 @@ let dispatch pipeline (type a) : a Query_protocol.t -> a =
     if path = "" then `Invalid_context else
       Locate.get_doc ~config
         ~env ~local_defs ~comments ~pos (`User_input path)
+  | Syntax_document pos ->
+    let typer = Mpipeline.typer_result pipeline in
+    let pos = Mpipeline.get_lexing_pos pipeline pos in
+    let node = Mtyper.node_at typer pos in
+    let res = Syntax_doc.get_syntax_doc pos node in 
+    (match res with
+    | Some res -> `Found res 
+    | None -> `No_documentation) 
+
 
```

Comment:
There is one empty line to many, and according to the rest of this file there should be an empty line before instead.